common.groovy

```diff
@@ -56,7 +56,7 @@ defaults = [
   ],
   maxBuildsPerNode: 1,
   maxTotalConcurrentBuilds: 3,
-  maxWorkflowTestConcurrentBuilds: 3,
+  maxWorkflowTestConcurrentBuilds: 13,
```

Comment:
if this is meant for `workflow-test-pr`, I believe we would bump the value immediately below this one.  (This one I believe is for `workflow-test`)